internal/trace/listener.go

```diff
@@ -134,3 +137,12 @@ func separateVersionFromFunctionArn(functionArn string) (arnWithoutVersion strin
 	}
 	return arnWithoutVersion, functionVersion
 }
+
+func isLambdaMode(extensionPath string) bool {
```

Comment:
Maybe we could consolidate with this logic https://github.com/DataDog/datadog-lambda-go/blob/main/internal/metrics/listener.go#L98